dev-python/milksnake/milksnake-0.1.5.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{5,6,7})
```

Comment:
I know it's been sitting for a while, but could you rebase against fresh ::gentoo?

What I wanted to ask: Is python2.7 necessary? We are looking into removing it soon. Also if possible, test with python3.8. Deps look like it could work.

Reply:
Py2 is not necessary, I'll remove it and rebase.